app/ModelSerializers/Summit/Registration/SummitAttendeeSerializer.php

```diff
@@ -45,7 +45,8 @@ class SummitAttendeeSerializer extends SilverStripeSerializer
         'ticket_types',
         'allowed_access_levels',
         'allowed_features',
-        'tags'
+        'tags',
+        'notes'
```

Comment:
dont do this
this is a public one
please do check main
its already implemented at private one